test/Graphics/Drawable.test.cpp

```diff
@@ -7,3 +10,40 @@ static_assert(!std::is_copy_constructible_v<sf::Drawable>);
 static_assert(std::is_copy_assignable_v<sf::Drawable>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Drawable>);
 static_assert(std::is_nothrow_move_assignable_v<sf::Drawable>);
+
+class DrawableTest : public sf::Drawable
+{
+public:
+    void draw(sf::RenderTarget&, const sf::RenderStates&) const override
+    {
+        m_called += 1;
+    }
+    int callCount() const
+    {
+        return m_called;
+    }
+
+protected:
+    mutable int m_called{};
+};
+
+TEST_CASE("[Graphics] sf::Drawable")
```

Comment:
Go look at the `sf::Texture` tests to see how we skip display-dependent tests. There's some special code you need to add to this `TEST_CASE` call.